templates/components/common/body.html

```diff
@@ -9,4 +9,5 @@
     </div>
     {{> components/common/modal/modal}}
     {{> components/common/alert/alert-modal}}
+    <div class="c-body-overlay"></div>
```

Comment:
I know it's just one line, but would be good to export it to a separate file, as it's a separate component